### ISSUE TITLE
Bad RPC error messages on init

### ISSUE THREAD

**Issue by torkelrogstad** [ordinals/ord#3064]
When doing for example `ord index update` or `ord wallet inscriptions` while my Bitcoin Core node is turned off, I get this:

```
$ ord wallet inscriptions
error: cookie file `/Users/torkel/Library/Application Support/Bitcoin/.cookie` does not exist
```

The inexperienced user would most likely not realize that this error message means "your node is turned off, and therefore there exists no way to authenticate with it. Please turn if on and try again". 

An obvious improvement would be to add a simple prefix to the error message, something like `could not initialize RPC: <msg>`. Perhaps we could even add something like `the node is turned off, or you misconfigured the node connection settings`?

**Comment by so7ow** [comment]
That's a common failure message even when the node is turned on. If RPC creds are specified on Bitcoin but not on ord, or if RPC server is disable are two scenarios where you'll get that message even with the node running.

**Comment by torkelrogstad** [comment]
Right, good point. Might be a good idea to stick to the simple version, then! (i.e. `could not initialize RPC: <msg>`)
